src/content-processor/markdown-process.js

```diff
@@ -139,15 +139,15 @@ const markedProcessorFactory = ({
           // For HTML render
           const imageId = token.href.split('/').pop();
           const imageExt = window.contentConfig.images[imageId];
-          return `<img src="./images/${imageExt}" alt="${token.text}">`;
+          return `<img src="./images/${imageExt}" alt="${token.text}" data-id="${imageId}">`;
```

Comment:
Could we have a specific attribute name instead of data-id? For example, data-seemark-image-id